js/ace.js

```diff
@@ -0,0 +1,1034 @@
+        const ticker = this.safeValue (response, marketId, {});
+        //
+        //     {
+        //         "BTC/USDT":{
+        //             "base_volume":229196.34035399999,
+        //             "last_price":11881.06,
+        //             "quote_volume":19.2909
+        //         }
+        //     }
+        //
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @url https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#oapi-api---trade-data
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.publicGetOapiV2ListTradePrice ();
+        //
+        //     {
+        //         "BTC/USDT":{
+        //             "base_volume":229196.34035399999,
+        //             "last_price":11881.06,
+        //             "quote_volume":19.2909
+        //         }
+        //     }
+        //
+        const tickers = [];
+        const pairs = Object.keys (response);
+        for (let i = 0; i < pairs.length; i++) {
+            const marketId = pairs[i];
+            const market = this.safeMarket (marketId);
+            const rawTicker = this.safeValue (response, marketId);
+            const ticker = this.parseTicker (rawTicker, market);
+            tickers.push (ticker);
+        }
+        return this.filterByArray (tickers, 'symbol', symbols);
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchOrderBook
+         * @description fetches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
+         * @url https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#open-api---order-books
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int|undefined} limit the maximum amount of order book entries to return
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-book-structure} indexed by market symbols
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
```

Comment:
@sc0Vu aren't we missing the depth/limit here?
![image](https://user-images.githubusercontent.com/43336371/213666058-7423a6e2-a85a-41c6-9d2e-30293b337fe0.png)